CAP/gap/CategoryMorphisms.gi

```diff
@@ -572,6 +572,62 @@ end );
 #     
 # end );
 
+##
+InstallOtherMethod( PreComposeList,
+               [ IsCapCategory, IsList ],
+               
+  function( cat, morphism_list )
+    
+    if IsEmpty( morphism_list ) then
+        Error( "the give list of morphisms is empty\n" );
```

Comment:
```suggestion
        Error( "the given list of morphisms is empty\n" );
```
This typo occurs three more times below.

Reply:
Thanks, done.